src/ui/views/TradingBlock.tsx

```diff
@@ -6,6 +6,10 @@ import { getCols, helpers, toWorker } from "../util";
 import { DataTable, PlayerNameLabels } from "../components";
 import type { View, ThenArg } from "../../common/types";
 import type api from "../../worker/api";
+import TradeFilter, { filterType } from "../views/TradeFilter/TradeFilter";
+import { AnimatePresence, motion } from "framer-motion";
+import FilterItem from "../../worker/core/trade/filterItem";
+import _ from "lodash";
```

Comment:
For bundle size concerns, do it like `import intersection from "lodash/intersection";`. Otherwise, not sure if my build pipeline is smart enough to ignore the rest of lodash.